packages/lumx-react/src/components/message/Message.tsx

```diff
@@ -0,0 +1,126 @@
+import { mdiAlert, mdiAlertCircle, mdiCheckCircle, mdiInformation } from '@lumx/icons';
+import { ColorPalette, Icon } from '@lumx/react';
+import { COMPONENT_PREFIX, CSS_PREFIX } from '@lumx/react/constants';
+import { IGenericProps, getRootClassName, handleBasicClasses } from '@lumx/react/utils';
+import classNames from 'classnames';
+import React, { ReactElement } from 'react';
+
+/////////////////////////////
+
+enum KindMessage {
+    error = 'error',
+    info = 'info',
+    success = 'success',
+    warning = 'warning',
+}
```

Comment:
"KindMessage" makes it sound like the message is kind xD
Maybe we could re-name it "MessageKind"?

Reply:
`NiceChat`, `PepTalk`, `InspirationQuote`, `WellSaid`